app/app.component.js

```diff
@@ -0,0 +1,10 @@
+(function(app) {
+    app.AppComponent =
+        ng.core.Component({
+            selector: 'my-app',
+            template: '<h1>Heptonstall Weather Webcam</h1>'
+        })
+        .Class({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://www.codacy.com/app/garethjackson/www-heptonstall-info/file/2734059926/issues/source?bid=3331202&fileBranchId=3351007#l7)